<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo52</class>
 <widget class="QDialog" name="dlgDemo52">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mantenimiento de Productos dentro del QTableView</string>
  </property>
  <widget class="QLabel" name="lblTotal">
   <property name="geometry">
    <rect>
     <x>1154</x>
     <y>20</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRegistros">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total de Registros:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblProducto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1181</width>
     <height>521</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNuevo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
